utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in

```diff
@@ -122,6 +123,11 @@ Upgrade to the latest major version. For example, if you are running
 version **3.2** then this could upgrade you to **3.3**.
 
 
+UPGRADE
```

Comment:
Also, make sure you re-run `make docs` from the `utils` directory and commit the changed `.1` file.